packages/nextra/styles/code-block.css

```diff
@@ -71,7 +71,7 @@ pre code.nextra-code:not([class*='twoslash-']) {
   --twoslash-popup-shadow: rgba(0, 0, 0.08) 0px 1px 4px;
   --twoslash-docs-color: #888;
   --twoslash-docs-font: sans-serif;
-  --twoslash-code-font: inherit;
+  --twoslash-code-font: ui-monospace;
```

Comment:
Great PR, can we change the current font? I feel it doesn't look great 
<img width="774" alt="image" src="https://github.com/user-attachments/assets/e46a4a2d-d2d4-42e3-8477-502d98c958de">

Reply:
Oh.. you are right. It dosen't look good 😨 

Previously, the popup element was inside the pre > code, so when I applied inherit, the font was applied as the one used in the code block. To fix this, I specified `ui-monospace`, but It doesn't fixed.. 😢 

What do you think about applying this instead?

```
ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
```

This is the font-mono from Tailwind CSS.
https://tailwindcss.com/docs/font-family

Reply:
I would like to have the same font as the code block has, wdyt?

Reply:
Yes, I agree! 
I thought font-mono was for the code in the code block—am I wrong?